client/extensions/woocommerce/app/settings/payments/payments-location-currency.js

```diff
@@ -1,17 +1,36 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import AddressView from '../../../components/address-view';
 import Card from 'components/card';
+import FormSelect from 'components/forms/form-select';
 import ExtendedHeader from '../../../components/extended-header';
 
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getPaymentCurrencySettings } from '../../../state/wc-api/settings/general/selectors';
+import { fetchSettingsGeneral } from '../../../state/wc-api/settings/general/actions';
+
 class SettingsPaymentsLocationCurrency extends Component {
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+		currency: PropTypes.object,
```

Comment:
Let's show the shape of this prop -- at least the fields you are using (`value` and `options`).